ledger-core-ethereum/src/ethereum/database/Migrations.cpp

```diff
@@ -2,9 +2,9 @@
 
 namespace ledger {
     namespace core {
-        int const EthereumMigration::coinID = 60;
```

Comment:
Maybe we can put here some preprocessor macros here to check the C++ version - in case we upgrade our version later ? In C++17 we don't need this declaration anymore and more than that it's deprecated.

Reply:
We’re not using C++17 so why bother now? When we move to C++17 (if we ever do) that line will make a compile error and we’ll fix it.